bin/pymultimonaprs

```diff
@@ -91,8 +91,15 @@ def bc():
 		sleep(config['beacon']['send_every'])
 
 logger.info("Starting pymultimonaprs")
+				
+if 'prefered_protocol' in config:
+	prefered_protocol = config['prefered_protocol'].lower()
```

Comment:
You should use config.get("prefered_protocol", DEFAULTVALUE) to avoid breaking the configparsing for users without the entry in their config file.